src/main/java/hudson/tasks/junit/pipeline/JUnitResultsStep.java

```diff
@@ -124,7 +124,7 @@ public StepExecution start(StepContext context) throws Exception {
         return new JUnitResultsStepExecution(this, context);
     }
 
-    @OptionalExtension(requirePlugins = {"pipeline-stage-step","workflow-cps","workflow-job"})
+    @Extension
```

Comment:
Must always be enabled. Otherwise there is no `junit` step on installations without `pipeline-stage-step`! Ouch.

Reply:
should have just removed the pipeline-stage step then.